package.json

```diff
@@ -20,25 +20,56 @@
     "url": "https://github.com/mjbvz/vscode-github-markdown-preview-style/issues"
   },
   "engines": {
-    "vscode": "^1.15.0"
+    "vscode": "^1.41.0"
   },
   "categories": [
     "Other"
   ],
+  "activationEvents": [
```

Comment:
These should not be required. VS Code's markdown support will activate this through `markdown.markdownItPlugins`

Reply:
Done in ebe7d18